api/lambdas/cda-to-visualization/index.js

```diff
@@ -0,0 +1,217 @@
+const puppeteer = require("puppeteer-core");
+const chromium = require("@sparticuz/chromium");
+const AWS = require("aws-sdk");
+const SaxonJS = require("saxon-js");
+const fs = require("fs");
+const styleSheetText = require("./stylesheet.js");
+
+const getEnvOrFail = name => {
+  const value = process.env[name];
+  if (!value || value.trim().length < 1) throw new Error(`Missing env var ${name}`);
+  return value;
+};
+
+const bucketName = getEnvOrFail("MEDICAL_DOCUMENTS_BUCKET_NAME");
+
+const s3client = new AWS.S3({
+  signatureVersion: "v4",
+});
+
+module.exports = async req => {
```

Comment:
Missing Sentry here, either on the handler and/or with a catch-all around the code

Reference: https://github.com/metriport/metriport/blob/develop/api/lambdas/sqs-to-fhir/index.js#L69